google/cloud/bigtable/examples/data_filter_snippets.cc

```diff
@@ -494,35 +479,55 @@ void InsertTestData(google::cloud::bigtable::Table table,
   // the actual examples during the CI builds.
```

Comment:
It might be helpful to add a comment here about where the data came from. Something like:
```c++
  // the actual examples during the CI builds.
  // 
  // The data is from: https://cloud.google.com/bigtable/docs/using-filters#data
```